cd/scripts/release-controller.sh

```diff
@@ -143,9 +145,8 @@ if ! buildah bud \
   --build-arg service_controller_git_commit="$SERVICE_CONTROLLER_GIT_COMMIT" \
   --build-arg build_date="$BUILD_DATE" \
   --build-arg golang_version="$GOLANG_VERSION" \
-  --build-arg target_arch="linux/arm64" \
-  --arch "arm64" \
-  --variant v8 \
+  --build-arg target_arch="arm64" \
+  --arch "amd64" \
```

Comment:
@kahirokunn, your initial approach is correct. However, we need to specify the `buildah` architecture as amd64. This is necessary because the base images we use for building ACK binaries do not rely on golang images; instead, we download Go compiler binaries in the amd architecture. While we might consider making broader changes in the future, for now let's go with this simple fix.

See: https://github.com/aws-controllers-k8s/test-infra/blob/main/prow/jobs/images/Dockerfile.deploy#L30

Reply:
Okay, I see what you are talking about with this commit .

292ed578889d37f48c3d09cf62b6e98fbc72ca9d

Thank you for your response!